README.md

```diff
@@ -19,10 +19,9 @@
 mobile heterogeneous computing platforms. The design focuses on the following
 targets:
 * Performance
-  * The runtime is highly optimized with NEON, OpenCL and Hexagon, and
+  * Tuntime is optimized with NEON, OpenCL and Hexagon, and
```

Comment:
Tuntime?

Reply:
@JamesBie You should amend your commit and force push to this `README` branch.

Reply:
@l2dy  i've amended my commit, pushed and submitted another pull request

Reply:
@JamesBie Thanks for the patch.
Is this deprecated by https://github.com/XiaoMi/mace/pull/34?
Actually, you can use `git rebase HEAD~2` to squash top 2 commits and force to push to `JamesBie:README` branch.

Reply:
@llhe  Yes its depreciated by #34 so you could close this. Thanks for the tip! I'll be sure to do that next time and sorry about that!